students/chieu_quach/lesson04/mailroom_part2.py

```diff
@@ -43,15 +41,16 @@
       "\n\n"
       )
 
+from operator import *
```

Comment:
Make sure import statements are at the top of the file.  Also, you very rarely want to use the star wildcard when importing.